sql/core/src/test/resources/sql-tests/results/udtf/udtf.sql.out

```diff
@@ -335,6 +335,37 @@ org.apache.spark.sql.AnalysisException
 }
 
 
+-- !query
+SELECT * FROM UDTFPartitionByOrderByComplexExpr(TABLE(t2))
+-- !query schema
+struct<partition_col:int,count:int,total:int,last:int>
+-- !query output
+0	1	1	1
+1	2	5	3
+
+
+-- !query
+SELECT * FROM InvalidTestPythonUDTFOrderByAscKeyword(TABLE(t2))
+-- !query schema
+struct<>
+-- !query output
+org.apache.spark.sql.AnalysisException
+{
+  "errorClass" : "UNRESOLVABLE_TABLE_VALUED_FUNCTION",
+  "sqlState" : "42883",
+  "messageParameters" : {
+    "name" : "`InvalidTestPythonUDTFOrderByAscKeyword`"
```

Comment:
Is this expected? From the class name, it seems not expected?

Reply:
Apologies, I forgot to update the UDTF name. Fixed this now, the error message works as expected.

Reply:
Ah, then, the updated test seems to be duplicated now:

```sql
SELECT * FROM UDTFInvalidOrderByAscKeyword(TABLE(t2));
```

and the latter one failed with `ParseException` by missing `;` at the end.

Reply:
Thanks for pointing that out, fixed it again!